source/h2_frames.c

```diff
@@ -984,7 +984,7 @@ struct aws_h2_frame *aws_h2_frame_new_ping(
     bool writes_ok = true;
     writes_ok &= aws_byte_buf_write(&frame->encoded_buf, opaque_data, AWS_H2_PING_DATA_SIZE);
     AWS_ASSERT(writes_ok);
-
+    frame->base.high_priority = ack;
```

Comment:
Nice.
I was going to say we should make all ping frames be high priority, but the spec clearly says "PING **responses** SHOULD be given higher priority than any other frame"